AutoCollection/WebSnippet.ts

```diff
@@ -0,0 +1,146 @@
+import * as path from "path";
+import http = require("http");
+import fs = require('fs');
+
+import Logging = require("../Library/Logging");
+import TelemetryClient = require("../Library/TelemetryClient");
+
+class WebSnippet {
+
+    public static INSTANCE: WebSnippet;
+
+    private static _snippet: string;
+    private static _aiUrl: string;
+    private _isEnabled: boolean;
+    private _isInitialized: boolean;
+
+    constructor(client: TelemetryClient) {
+        if (!!WebSnippet.INSTANCE) {
+            throw new Error("Web snippet injection should be configured from the applicationInsights object");
```

Comment:
Not sure what this scenario is.  The error message should describe what they did wrong and what they should do. Is this to ensure we have a singleton object of WebSnippet?

Reply:
Yes! If the instance is not found, it means the auto injection config is set to false or some errors happening during init!